examples/server/json-schema-to-grammar.cpp

```diff
@@ -0,0 +1,734 @@
+#include "json-schema-to-grammar.h"
+#include <algorithm>
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <regex>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
+
+using json = nlohmann::json;
+using namespace std;
```

Comment:
Avoid `using namespace std;`

`using json = ...` is fine

Reply:
std::done